test/TestProjects/CollapseRequestCondition-LowLevel/CollapseRequestCondition-LowLevel.json

```diff
@@ -402,4 +565,4 @@
       }
     }
   }
-}
+}
```

Comment:
Nit: Add new line at the end of the file

Reply:
resolved